src/tsickle_compiler_host.ts

```diff
@@ -141,7 +141,6 @@ export class TsickleCompilerHost implements ts.CompilerHost {
       fileName: string, content: string, writeByteOrderMark: boolean,
       onError?: (message: string) => void, sourceFiles?: ts.SourceFile[]): void {
     if (path.extname(fileName) !== '.map') {
-      fileName = this.delegate.getCanonicalFileName(fileName);
```

Comment:
I took a look at tsc_wrapped, and it already does the canonicalization before writing.  But yes, definitely should test it first.